_nonmvptools/capetownbudgetproject.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: tool
-title: Cape Town Budget Project
+name: Cape Town Budget Project
```

Comment:
Also these are the tools we're not using now, I might delete them actually cause they don't add anything at this point.